### PR TITLE
Replace PR template

### PR DESCRIPTION
**Pull request by tengpengli**
Replace PR_TEMPLATE with a more descriptive one

The new template comes from Envoy github

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-- [ ] Tests pass
-- [ ] Appropriate changes to README are included in PR
\ No newline at end of file
+Commit Message:
+Additional Description:
+Risk Level:
+Testing:
+Docs Changes:
+Release Notes:
+[Optional Runtime guard:]
```
Comment (asraa) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Let's remove this and depcreated/release notes/risk level? These are not so relevant here

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
-> It's a good idea to open an issue first for discussion.
-
-- [ ] Tests pass
-- [ ] Appropriate changes to README are included in PR
\ No newline at end of file
+Commit Message:
+Additional Description:
+Risk Level:
+Testing:
+Docs Changes:
```
Comment (asraa) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: (and README)

**Comment by asraa** [comment]
Let's start with using the template for this :)

**Comment by tengpengli** [comment]
My bad. I should checkout from master rather than the launcher script. Let me delete this PR and restart a new one!

**Pull request closed without merging**